fe/fe-core/src/main/java/org/apache/doris/nereids/properties/UnboundLogicalProperties.java

```diff
@@ -37,4 +37,14 @@ public UnboundLogicalProperties() {
     public List<Slot> getOutput() {
         throw new UnboundException("getOutput");
     }
+
```

Comment:
ditto

Reply:
this method can help us find the bug quickly, if throw a exception